wp-content/themes/cpipr/partials/content-series.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/*
+ * The default template for displaying content
+ *
+ * @package Largo
+ */
+$args = array (
+	// post-specific, should probably not be filtered but may be useful
+	'post_id' => $post->ID,
+	'hero_class' => largo_hero_class( $post->ID, FALSE ),
+
+	// only used to determine the existence of a youtube_url
+	'values' => get_post_custom( $post->ID ),
+
+	// this should be filtered in the event of a term-specific archive
+	'featured' => has_term( 'homepage-featured', 'prominence' ),
+
+	// $show_thumbnail does not control whether or not the thumbnail is displayed;
+	// it controls whether or not the thumbnail is displayed normally.
+	'show_thumbnail' => TRUE,
+	'show_byline' => TRUE,
+	'show_excerpt' => TRUE,
+	'in_series' => FALSE,
+);
+
+$args = apply_filters( 'largo_content_partial_arguments', $args, get_queried_object() );
+
+extract( $args );
+
+$entry_classes = 'entry-content';
+
+$show_top_tag = largo_has_categories_or_tags();
+
+$authors = get_coauthors();
```

Comment:
if the function `get_coauthors()` isn't defined, then the template breaks and the page fails to finish loading: 

<img width="1200" alt="screen shot 2018-04-09 at 1 19 52 pm" src="https://user-images.githubusercontent.com/1754187/38512239-b9f5e74a-3bf8-11e8-8a82-0a875c8cfd76.png">

The parts of this template that are dependent upon the Co-Authors Plus plugin that defines `get_coauthors` should be wrapped in a conditional that checks if the function is defined before calling it, like so:

```php
if ( function_exists( 'get_coauthors' ) ) {
    $authors = get_coauthors();
} else {
    $authors = array();
    error_log( 'This theme depends upon Co-Authors Plus!' );
}
```

There are other places in the theme where `get_coauthors` and `coauthors_get_avatar` are called that should similarly be made safe.